app/controllers/claims/pages_controller.rb

```diff
@@ -1,2 +1,5 @@
 class Claims::PagesController < ApplicationController
+  def index
+    @school = User.first&.schools&.first&.gias_school
```

Comment:
```suggestion
    @school = current_user.schools
```
If the school needs additional information from the GiasSchool association delegate it.